src/main/java/io/fabric8/maven/docker/RemoveMojo.java

```diff
@@ -44,7 +44,7 @@
 public class RemoveMojo extends AbstractDockerMojo {
 
     // Should all configured images should be removed?
-    @Parameter(property = "docker.removeAll", defaultValue = "false")
+    @Parameter(property = "docker.removeAll", defaultValue = "false", alias = "removeAll")
```

Comment:
From the [Maven documentation](http://books.sonatype.com/mvnref-book/reference/writing-plugins-sect-mojo-params.html#writing-plugins-sect-param-annot) : 

>  The alias provides the name of the parameter. If alias is omitted, Maven will use the name of the variable as the parameter name. 

So this change is a noop since the field is called also `removeAll`.

In fact, you can already use `removeAll` as a plugin configuration parameter:

Either in a `<properties>` sections:

```xml
<properties>
   <docker.removeAll>true</docker.removeAll>
</properties>
```

or direct in the plugin configuration:

```xml
<plugin>
  <artifactId>docker-maven-plugin</artifactId>
  ...
  <configuration>
    <removeAll>true</removeAll>
    ....
  </configuration>
</plugin>
```

Please let me know whether this work for you. If not, its a bug.